SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -448,6 +448,8 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 	TArray<FWorkerComponentUpdate> ComponentUpdates =
 		UpdateFactory.CreateComponentUpdates(Object, Info, EntityId, RepChanges, HandoverChanges, OutBytesWritten);
 
+	TOptional<Trace_SpanId> CauseSpanId = EventTracer->PopLatentPropertyUpdateSpanId(Object);
```

Comment:
```if (EventTracer != nullptr && EventTracer->IsEnabled())```

Reply:
or is EventTracer always valid here, and will just return an empty span if disabled?

Reply:
The event tracer will be valid, and does handle return empty spanId if disabled. Think this is ok to leave for now.